packages/uploadthing/src/client.ts

```diff
@@ -169,7 +169,10 @@ export const DANGEROUS__uploadFiles = async <TRouter extends FileRouter>(
     // Poll for file data, this way we know that the client-side onUploadComplete callback will be called after the server-side version
     await pollForFileData(presigned.key);
 
+    // TODO: remove `file` prefix in next major version
     return {
+      fileName: file.name,
+      fileSize: file.size,
       fileKey: presigned.key,
       fileUrl: genUrl,
```

Comment:
Could also be a good idea to deprecate them now

```suggestion
    // @deprecated - use `name` instead
    fileName: file.name,
    name: file.name,
    // @deprecated - use `size` instead
    fileSize: file.size,
    size: file.size,
    // @deprecated - use `key` instead
    fileKey: presigned.key,
    key: presigned.key,
    // @deprecated - use `url` instead
    fileUrl: genUrl,
    url: genUrl,
```